react.md

```diff
@@ -5,40 +5,77 @@ Answer the following questions. First, without external resources. Challenge you
 
 1a. Indicate which of the following statements about React are false:
 
-- React is a modern, efficient answer to complex UI applications (true/false)
-- React will only render on the server using Node.js (true/false)
-- React is a full stack framework for modern web applications (true/false)
-- React is a flexible library that plays the role of V in an MVC framework (true/false)
-- You should always update a component's state directly using this.state (true/false)
-- React is made up of encapsulated components that manage their own state (true/false)
-- React components render HTML (true/false)
+1 - React is a modern, efficient answer to complex UI applications (true/false) 
+2 - React will only render on the server using Node.js (true/false) 
+3 - React is a full stack framework for modern web applications (true/false) 
+4 - React is a flexible library that plays the role of V in an MVC framework (true/false) 
+5 - You should always update a component's state directly using this.state (true/false) 
+6 - React is made up of encapsulated components that manage their own state (true/false) 
+7 - React components render HTML (true/false) true?
+
+- My Answers
+1 true
+2 true? 
+3 true?
+4 true? 
+5 false
+6 true?
+7 true?
+
+- Researched Answers
+1 true 
+2 false? 
+3 true
+4 true
+5 false
+6 true
+7 false
+
```

Comment:
This is false - Technically JSX is rendered within a React component.

Reply:
Thank you for the clarification!